blockchains/nativezetachain/info/info.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Native ZetaChain",
+    "website": "https://www.zetachain.com/",
+    "description": "ZetaChain is the only public, decentralized blockchain and smart contract platform that enables message passing and value transfer between any blockchain. Our omnichain smart contracts give the power of programmability to non-smart contract chains like Bitcoin and Dogecoin. No other blockchain enables fully interoperable smart contracts like we do.",
+    "explorer": "https://explorer.zetachain.com/cosmos/tx",
+    "research": "",
+    "symbol": "ZETA",
+    "type": "coin",
+    "decimals": 18,
```

Comment:
check decimals

Reply:
should be ok https://github.com/trustwallet/wallet-core/blob/e13420c9becb7e4b104e965d078f3519e719a0c6/registry.json#L4462